packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -191,20 +191,22 @@ export class TdsDropdown {
       nextElementSibling return the index for the first child in our Dropdown.  */
 
       const startingIndex = activeElement.nextElementSibling
-        ? this.children.findIndex((element) => element === activeElement.nextElementSibling)
```

Comment:
Do you know where the getChildren method is documented? 
Couldn't find any documentation on it.

Thinking it's possible it is expensive, so it might be better to run it once in the method, put the result in a variable, and use that.

Reply:
line 255 in dropdown.tsx:
  /* Returns a list of all children that are are tds-dropdown-option elements */
  private getChildren = () =>
    Array.from(this.host.children).filter((element) => element.tagName === 'TDS-DROPDOWN-OPTION');

Reply:
The problem with running it once it that we can't be sure all options are in the first render and so on

Reply:
Aha, I thought it was a built in Stencil method.

Once per method I mean.

Ie.

```js
 async onKeyDown(event: KeyboardEvent) {
    
    // ( ... )
    
    const children = this.getChildren(); // <---- Run once in onKeyDown
     
    if (event.key === 'ArrowDown') {
      
      // ( ... )

      const startingIndex = activeElement.nextElementSibling
        ? children.findIndex((element) => element === activeElement.nextElementSibling)
        : 0;

      const elementIndex = findNextFocusableElement(children, startingIndex);
      children[elementIndex].focus();
    } else if (event.key === 'ArrowUp') {
    
      // ( ... )
      
      const startingIndex = activeElement.nextElementSibling
        ? children.findIndex(
            (element) => element === activeElement.previousElementSibling,
          )
        : 0;

      const elementIndex = findPreviousFocusableElement(children, startingIndex);
      children[elementIndex].focus();
    } else if (event.key === 'Escape') {
      this.open = false;
    }
  }
  ```

Reply:
Makes sense! I'll update